assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -137,6 +162,12 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 	const currencyHeader = period.headers.find( ( header ) => null !== header.currency && 0 < header.currency.length );
 	const currencyCode = currencyHeader ? currencyHeader.currency : false;
 
+	const change = {
+		earnings: calculateAbsoluteChange( previousPeriod.totals[ 0 ], period.totals[ 0 ] ),
+		rpm: calculateAbsoluteChange( previousPeriod.totals[ 1 ], period.totals[ 1 ] ),
+		impressions: calculateAbsoluteChange( previousPeriod.totals[ 2 ], period.totals[ 2 ] ),
```

Comment:
I'm not sure the utility adds a benefit as-is. I like the idea but I think it ends up being clearer to do the subtraction here instead as it isn't obvious by the signature which parameter is current and which is previous. If it would be doing more then it would make sense to keep but as it is, it's basically just subtraction and casting numbers (which will happen automatically when subtracting if needed as well).